packages/taxdat/tests/testthat/test_setup_helpers.R

```diff
@@ -901,4 +899,21 @@ test_that("check_stan_debug works", {
   expect_error(check_stan_debug(debug = config_error$debug),
                "Debug needs to be logical") 
 })
+
+test_that("check_stan_model works", {
+  exist <- paste0("/home/usr/cholera_mapping_pipeline/Analysis/Stan/",
```

Comment:
We no longer have a Stan file of this name in the dev repo
Better to use this file name Analysis/Stan/mapping_model_inference.stan